selenium-system-test/src/test/scala/hts/driver/Driver.scala

```diff
@@ -62,8 +62,8 @@ object Driver {
   private val driverDirectory: String = "selenium-system-test/drivers"
 
   private def createChromeDriver(): WebDriver = {
-    if (Option(systemProperties.getProperty("webdriver.chrome.driver")).isEmpty) {
-      val _ = if (isMac) {
+    if (systemProperties.getProperty("webdriver.chrome.driver") == null) {
```

Comment:
We should avoid using `null` wherever possible - the wrapping of the return value in the `Option` and checking if the result is `None` is the proper way to work with `null` values in scala since
```scala
Option(null) = None
```
Avoiding `null` now will get you in the practice of not using it in the future. See this info here: https://alvinalexander.com/scala/scala-null-values-option-uninitialized-variables

Reply:
This would have been picked up by WartRemover but I had to disable the null checks because scalamock was giving false positives